src/main/java/WordFrequencyGame.java

```diff
@@ -1,75 +1,40 @@
-    public String getResult(String inputStr){
+    public String getResult(String inputStr) {
 
+        return inputStr.split("\\s+").length == 1 ? inputStr + " 1" : getCalculatedString(inputStr);
+    }
 
-        if (inputStr.split("\\s+").length==1) {
-            return inputStr + " 1";
-        } else {
-
-            try {
-
-                //split the input string with 1 to n pieces of spaces
-                String[] arr = inputStr.split("\\s+");
-
-                List<Input> inputList = new ArrayList<>();
-                for (String s : arr) {
-                    Input input = new Input(s, 1);
-                    inputList.add(input);
-                }
-
-                //get the map for the next step of sizing the same word
-                Map<String, List<Input>> map =getListMap(inputList);
-
-                List<Input> list = new ArrayList<>();
-                for (Map.Entry<String, List<Input>> entry : map.entrySet()){
-                    Input input = new Input(entry.getKey(), entry.getValue().size());
-                    list.add(input);
-                }
-                inputList = list;
-
-                inputList.sort((w1, w2) -> w2.getWordCount() - w1.getWordCount());
+    private String getCalculatedString(String inputStr) {
+        try {
 
-                StringJoiner joiner = new StringJoiner("\n");
-                for (Input w : inputList) {
-                    String s = w.getValue() + " " +w.getWordCount();
-                    joiner.add(s);
-                }
-                return joiner.toString();
-            } catch (Exception e) {
+            return getStrListMap(Arrays
+                    .stream(inputStr.split("\\s+"))
+                    .map(str -> new Input(str, 1))
+                    .collect(Collectors.toList()))
+                    .entrySet()
+                    .stream()
+                    .map(entry -> new Input(entry.getKey(), entry.getValue().size()))
+                    .sorted((w1, w2) -> w2.getWordCount() - w1.getWordCount())
+                    .map(input -> input.getValue() + " " + input.getWordCount())
+                    .collect(Collectors.joining("\n"));
+        } catch (Exception e) {
 
 
-                return "Calculate Error";
-            }
+            return "Calculate Error";
         }
     }
 
 
-    private Map<String,List<Input>> getListMap(List<Input> inputList) {
-        Map<String, List<Input>> map = new HashMap<>();
-        for (Input input :  inputList){
-//       map.computeIfAbsent(input.getValue(), k -> new ArrayList<>()).add(input);
-            if (!map.containsKey(input.getValue())){
-                ArrayList arr = new ArrayList<>();
-                arr.add(input);
-                map.put(input.getValue(), arr);
-            }
-
-            else {
-                map.get(input.getValue()).add(input);
-            }
-        }
-
-
-        return map;
+    private Map<String, List<Input>> getStrListMap(List<Input> inputList) {
```

Comment:
This function can be replace with one line using stream, by collecting it to 
.collect(Collectors.groupingBy(word -> word))

also, since we are getting the size ultimately, suggest
.collect(Collectors.groupingBy(Function.identity(), summingInt(x -> 1)))
ti directly get size